docs/geedocs/answer/172794.html

```diff
@@ -19,6 +19,5 @@
 <p>When you install GEE on a server, it queries for the name of the local host by issuing a <code>hostname -f</code> command. Each time you build a resource, project, or database on the server, the host name is added to all the files in each asset’s subdirectory. If you change the hostname or if you move assets to another server, you must also update the <code>assetroot host</code> entry to match the new host name.</p>
 <p>To change the asset root host entry, use the <code>geconfigureassetroot</code> command:</p>
 <code>sudo /opt/google/bin/geconfigureassetroot --fixmasterhost</code>
-<p>For more information about the <code>geconfigureassetroot</code> command, see the <em>Command Line Reference</em> chapter of the <em>Google Earth Enterprise Reference Guide.</em></p>
```

Comment:
I'm not aware of any place that it's publicly available, so the easiest solution was to just remove the reference.